observability/docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>Observability</title>
+<productname>Red Hat OpenShift Service Mesh</productname>
+<productnumber>3.0</productnumber>
+<subtitle>Observability and Service Mesh</subtitle>
+<abstract>
+    <para>This document provides an overview of Red Hat OpenShift Observability integrations with OpenShift Service Mesh.
```

Comment:
Not sure how to see this with a D.O.C preview link. This is more of a customer portal update?

Reply:
This is for Pantheon purposes so there is no preview.